cpp/unreal_plugins/SimulationController/Source/SimulationController/UrdfBotAgent.cpp

```diff
@@ -5,6 +5,8 @@
 #include "SimulationController/UrdfBotAgent.h"
 
 #include <Components/SceneCaptureComponent2D.h>
+#include <GameFramework/PlayerStart.h>
```

Comment:
I cherry-picked this C++ code out of this PR, extended it to all of our agents, and slightly modified it (the config keys are slightly different), so we could start building public release executables for the upcoming release ASAP. This PR can be Python-only.